docker-compose.yaml

```diff
@@ -73,3 +73,14 @@ services:
       environment:
          <<: *common-config
       command: ["ozone","s3g"]
+   httpfs:
+     <<: *image
+     ports:
+       - 14000:14000
```

Comment:
nit: the rest of this file has indentation of 3 spaces, please use the same for the new service.